src/content/docs/kubernetes-pixie/k8s-otel/install.mdx

```diff
@@ -0,0 +1,635 @@
+    * [Kubelet receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver)
+    * [MetricsTransform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor)
+    * [Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver)
+    * [ResourceDetection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor)
+    * [Resource processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourceprocessor)
+    * [Transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor)
+
+    </Collapser>
+
+</CollapserGroup>
+
+
+### Supported Environments [#supported-envs]
+
+Our OpenTelemetry monitoring for Kubernetes provides robust support across various deployment environments. Supported platforms include:
+
+* **Cloud vendors:**
+
+  * Amazon EKS
+  * Microsoft AKS
+  * Google GKE
+  * Red Hat OpenShift
+
+* **On-Premise clusters:** We offer support for on-premise Kubernetes clusters.
+* **Kubernetes versions:** Support aligns with the Kubernetes versions currently supported by each vendor, ensuring compatibility and effective monitoring solutions across these environments.
+
+<Callout variant="tip">
+
+    Windows nodes are not supported.
+
+</Callout>
+
+
+## Install your Kubernetes cluster with OpenTelemetry [#install]
+
+You can install OpenTelemetry for Kubernetes using one of the following methods:
+
+
+<CollapserGroup>
+
+    <Collapser
+        id="helm-install"
+        title="Helm install"
+    >
+
+The Helm installation method is the recommended approach for deploying OpenTelemetry on Kubernetes. You can use the NRDOT or your own OpenTelemetry Collector. This method simplifies the deployment process and allows for easy configuration management.
+
+New Relic recommends using the NRDOT for Kubernetes, which is pre-configured to work seamlessly with New Relic. This distribution includes all necessary components and configurations to monitor your Kubernetes cluster effectively. However, if you prefer to use your own OpenTelemetry Collector, you can do so by ensuring it includes the required components for Kubernetes monitoring.
+
+### Configure using NRDOT [#configure-nrdot]
+
+1. Download the [Helm chart values file](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/values.yaml#L20-L24) adapt it to meet your specific requirements. For more information on the parameters, refer to the [parameters in `value.yaml` file](#config-params).
+
+   * Cluster name and <InlinePopover type="licenseKey"/> are mandatory.
+
+   * Check the entire list of [configuration parameters](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector#values).
```

Comment:
I would say something similar to : 
"If you want to know more about the other parameters......."

And instead of redirecting them to GitHub I would redirect them to your own section where you have the parameters listed